packages/vscode-extension/src/service/ImportObjectService.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ ** Copyright (c) 2021 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+import SuiteCloudRunner from '../core/SuiteCloudRunner';
+import DummyConsoleLogger from '../loggers/DummyConsoleLogger';
+import VSConsoleLogger from '../loggers/VSConsoleLogger';
+import { ConsoleLogger } from '../util/ExtensionUtil';
+import MessageService from './MessageService';
+import { IMPORT_OBJECTS, LIST_OBJECTS } from './TranslationKeys';
+import { VSTranslationService } from './VSTranslationService';
+
+const IMPORT_OBJECT_COMMAND_NAME = 'object:import';
+const LIST_OBJECT_COMMAND_NAME = 'object:list';
+const CONSOLE_LOGGER_ERROR = 'vsConsole Logger not initialized';
+
+export default class CustomObjectService {
+	private executionPath?: string;
+	private readonly messageService: MessageService;
+	private vsConsoleLogger: VSConsoleLogger | undefined;
+	private translationService: VSTranslationService;
+
+	constructor(messageService: MessageService, translationService: VSTranslationService) {
+		this.messageService = messageService;
+		this.translationService = translationService;
+	}
+
+	async importObjects(
+		destinationFolder: string,
+		appId: string | undefined,
+		scriptIds: string[] | undefined,
+		includeReferencedFiles: boolean,
+		executionPath: string | undefined
+	) {
+		this.executionPath = executionPath;
+		//We choose 'ALL' types because it is chosen before which scriptIds should be imported and not by type
+		let commandArgs: any = { type: 'ALL', destinationfolder: destinationFolder };
+
+		commandArgs.scriptid = scriptIds && scriptIds.length > 0 ? scriptIds : 'ALL';
+
+		if (appId == undefined || appId.length != 0) {
```

Comment:
We should use === instead of == unless there is really a need to use ==. Please check all other == usage in here and change them to ===.